packages/website/docs/components/ai_patterns/AI QuickPrompt/main.css

```diff
@@ -0,0 +1,32 @@
+ui5-card {
+	width: 600px;
```

Comment:
The sample should be under "patterns", not "ai_patterns" - you can just reuse the existing mdx and build upon it.